src/app/components/ProseMIrror/schema.ts

```
@@ -14,6 +14,72 @@ export const textSchema = new Schema({
     },
+    heading: {
+      attrs: { level: { default: 1 } },
+      content: "inline*",
+      group: "block",
+      defining: true,
+      parseDOM: [
+        { tag: "h1", attrs: { level: 1 } },
+        { tag: "h2", attrs: { level: 2 } },
+        { tag: "h3", attrs: { level: 3 } }
+      ],
+      toDOM(node) { return ["h" + node.attrs.level, 0] }
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)